tests/admin_widgets/tests.py

```diff
@@ -468,7 +468,7 @@ def test_readonly_fields(self):
 class ForeignKeyRawIdWidgetTest(TestCase):
 
     def test_render(self):
-        band = models.Band.objects.create(name='Linkin Park')
+        band = models.Band.objects.create(pk=1, name='Linkin Park')
```

Comment:
The problem is that different engines behave differently, so the auto-generated key varies per-engine, hence, there's no obvious value to expect below (line 484).

Maybe I should modify the below expected value, and format the pk into the expected string? Would that be better?